x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -224,7 +225,6 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
     loading,
     { events, updatedAt, loadPage, pageInfo, refetch, totalCount = 0, inspect },
   ] = useTimelineEvents({
-    alertConsumers,
```

Comment:
Don't we need alertConsumers for the RBAC request here?

Reply:
Just waiting on @yctercero here and will approve. Everything works great!

Reply:
we do need that for the RBAC, was there a reason for removing it?

Reply:
Fixed here: https://github.com/elastic/kibana/pull/108092/commits/36680b4d1ed8df78b73bd25745ae1aae194c7a1a